user/pages/03.Howtos/15.repartition-vm-disk/docs.en.md

```diff
@@ -315,6 +208,115 @@ root@partition-test:~# blkid /dev/vda2
+(So you might also use this approach to setup a LVM)
+
+```shell
+Hex code or GUID (L to show codes, Enter = 8300): L
+0700 Microsoft basic data  0c01 Microsoft reserved    2700 Windows RE
+3000 ONIE boot             3001 ONIE config           3900 Plan 9
+4100 PowerPC PReP boot     4200 Windows LDM data      4201 Windows LDM metadata
+4202 Windows Storage Spac  7501 IBM GPFS              7f00 ChromeOS kernel
+7f01 ChromeOS root         7f02 ChromeOS reserved     8200 Linux swap
+8300 Linux filesystem      8301 Linux reserved        8302 Linux /home
+8303 Linux x86 root (/)    8304 Linux x86-64 root (/  8305 Linux ARM64 root (/)
+8306 Linux /srv            8307 Linux ARM32 root (/)  8400 Intel Rapid Start
+8e00 Linux LVM             a000 Android bootloader    a001 Android bootloader 2
+a002 Android boot          a003 Android recovery      a004 Android misc
+a005 Android metadata      a006 Android system        a007 Android cache
+a008 Android data          a009 Android persistent    a00a Android factory
+a00b Android fastboot/ter  a00c Android OEM           a500 FreeBSD disklabel
+a501 FreeBSD boot          a502 FreeBSD swap          a503 FreeBSD UFS
+a504 FreeBSD ZFS           a505 FreeBSD Vinum/RAID    a580 Midnight BSD data
+a581 Midnight BSD boot     a582 Midnight BSD swap     a583 Midnight BSD UFS
+a584 Midnight BSD ZFS      a585 Midnight BSD Vinum    a600 OpenBSD disklabel
+a800 Apple UFS             a901 NetBSD swap           a902 NetBSD FFS
+a903 NetBSD LFS            a904 NetBSD concatenated   a905 NetBSD encrypted
+a906 NetBSD RAID           ab00 Recovery HD           af00 Apple HFS/HFS+
+af01 Apple RAID            af02 Apple RAID offline    af03 Apple label
+```
+
+After provisioning we may for our new created 40 GB partition inside of the VM :
+
+```shell
+ubuntu@partition-test:~$ df -h
+Filesystem      Size  Used Avail Use% Mounted on
+udev            984M     0  984M   0% /dev
+tmpfs           200M  644K  199M   1% /run
+/dev/vda1       9.6G  1.1G  8.5G  12% /
+tmpfs           997M     0  997M   0% /dev/shm
+tmpfs           5.0M     0  5.0M   0% /run/lock
+tmpfs           997M     0  997M   0% /sys/fs/cgroup
+/dev/vda15      105M  6.6M   98M   7% /boot/efi
+/dev/vda2        40G   49M   38G   1% /mnt/disks/data
+tmpfs           200M     0  200M   0% /run/user/1000
+```
+
 ### Conclusion
 
-We have started a VM using custom user data which prevents the growing of the VM root partition to the complete size of the ephemeral disk. Afterwards we have used the resulting free disk space to create a custom partition. In the end we have created a new ext4 filesystem on the new partition and mounted it so we can use it.
+We have started a VM using custom user data which prevents the growing of the VM root partition to the complete size of the ephemeral disk. The resulting free disk space was used to setup custom partitions on which we created an ext4 filesystem, mounted it and made our changes persistent for reboots.
```

Comment:
Aren't partition and filesystem changes always "persistent for reboots"? Speaking of reboots, I'm not sure what happens in variant #2 when the system gets rebooted multiple times and the space is already used.

Reply:
I think the mount is not persistent, I will give it some more testing and revisit the text afterwards

Reply:
Cloudinit is just executed once at VM creation. Thus I do not see this being a problem. Only if you refer to instance / volume snapshots I get the point. Will add a warning about that. Partition and Filesystem changes are persistent yes, but mount is not, updating the text accordingly =)